src/cert/mod.rs

```diff
@@ -109,6 +114,168 @@ pub async fn upload_new_cert_ref(
     Ok(cert_ref)
 }
 
+fn format_cert_for_multi_select(cert: &CageSigningCert) -> String {
+    let name = cert.name().unwrap_or_else(|| "".to_string());
+    let cert_hash = cert.cert_hash();
+    let not_after = cert
+        .not_after()
+        .and_then(|time| format_expiry_time(&time).ok())
+        .unwrap_or_else(|| "".to_string());
```

Comment:
Think an empty cell would be confusing, better to be explicit for the edge case:
```suggestion
        .unwrap_or_else(|| "Failed to get cert expiry".to_string());
```